synapseAnnotations/data/experimentalData.json

```diff
@@ -435,6 +445,11 @@
         "source": ""
       },
       {
```

Comment:
Agree with @allaway here, epigenetics is so vague I wasn't allowed to use it in a manuscript title (that talked about two various forms of epigenetic activity) - please stick to chromatin activity.

Reply:
Agreed :)